.changeset/automated-from-c2dd82ec06831a846c3d7d7e47261a04b2b4f6a3.md

```diff
@@ -0,0 +1,14 @@
+---
+# each of these should be one of "patch", "minor", "major"
+"@embroider/compat": TODO
+"@embroider/core": TODO
```

Comment:
remove, https://github.com/embroider-build/embroider/pull/1340/files#diff-a5cbba44c120a6bcfc99a5a7fe32bc73be243236913f8060889d56a02e4a79ff does not change this package, and changesets will bump consumers appropriately